src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java

```diff
@@ -15,6 +15,7 @@
  */
 package com.arpnetworking.commons.builder;
 
+import com.arpnetworking.commons.builder.annotations.WovenValidation;
```

Comment:
Should we update line 82 to specify it's only needed as a plugin dependency?

Reply:
Noted